examples/test_performance/clear_files.py

```diff
@@ -0,0 +1,27 @@
+#this script resets the data.txt and example_nf_deploy scripts and returns them to their original state
```

Comment:
if you modify the above script so it doesn't modify the base template, then this file might not be needed anymore.

Reply:
Any resources or ideas on how to do that? I'm very unfamiliar and have no formal training with shell scripting

Reply:
Start by making your change-NFs.py script so it generates an entirely new file instead of modifying the existing template. That shouldn't take a very big change -- you just need a loop that keeps adding segments to the file instead of only adding one segment. Once you have that working we can rethink how the other files interact.